src/commands/maps/remove_m_from_pool.ts

```diff
@@ -0,0 +1,42 @@
+import {ApplicationCommandOptionTypes} from "discord.js/typings/enums";
+import {ICommand} from "wokcommands";
+
+import {removeFromPool} from "../../abstract_commands/maps/remove_from_pool";
+import {ensure} from "../../utils/general";
+
+export default {
+    category: "Admin",
+    description: "Remove a map from a pool",
+
+    slash: true,
+    testOnly: true,
+    guildOnly: true,
+
+    options: [
+        {
+            name: "pool_name",
```

Comment:
use UUIDs here

make the pool name optional, if unspecified remove it from all the pools that have it (but do not delete the map)

Reply:
Resolved.